components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/fetch/GeckoViewFetchClient.kt

```diff
@@ -107,11 +108,17 @@ private fun WebRequest.Builder.addBodyFrom(request: Request): WebRequest.Builder
     return this
 }
 
-private fun WebResponse.toResponse(): Response {
+@VisibleForTesting
+@Suppress("MagicNumber")
+internal fun WebResponse.toResponse(): Response {
     val headers = translateHeaders(this)
+    val isBlobUrl = uri.toUri().scheme == "blob"
```

Comment:
Nit: I find it a little counter-intuitive that 'toUri()' is being called on a variable called 'uri'; my instinct is to wonder 'what is uri then if not a Uri'?

Reply:
Yes, it's a bit odd. It's part of the GeckoView api [WebMessage.uri](https://dxr.mozilla.org/mozilla-central/source/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/WebMessage.java#30) this variable is exposed a string.

Reply:
No worries then, just something to note for the future :)

Reply:
👍